src/app/components/AssignmentDutyCard/AssignmentDutyCard.tsx

```diff
@@ -46,6 +50,9 @@ export default class AssignmentDutyCard extends React.PureComponent<AssignmentDu
                 <AssignmentDutyActionsPanel>
                     <AssignmentDutyEditModal color={style.color} dutyId={id} />
                 </AssignmentDutyActionsPanel>
+                <AssignmentDutyInformationPanel>
+                    {hasComments && <Glyphicon glyph="comment" style={{fontSize: 16, paddingTop: 10}}/>}
```

Comment:
Perhaps set the title of the glyphicon to the comment so that if they chose they could read it without having to open up the whole sheriff card.